src/components/Editor/Breakpoints.js

```diff
@@ -54,6 +53,8 @@ class Breakpoints extends Component<Props> {
 }
 
 export default connect(state => ({
-  breakpoints: getVisibleBreakpoints(state),
+  // Retrieves only the first breakpoint per line so that the
+  // breakpoint marker represents only the first breakpoint
+  breakpoints: getFirstVisibleBreakpoints(state),
```

Comment:
`getFirstVisibleBreakpoint`

Reply:
Yeah, the naming here is a bit hard. `getFirstVisibleBreakpoint` seems to imply it returns only one breakpoint.  Maybe `getFirstVisibleBreakpointPerLine`?

Reply:
or `getFirstVisibleBreakpointOnLine` 

per feels like for each line

Reply:
good call

Reply:
good call